### ISSUE TITLE
[C2] Global questions bank update

### ISSUE THREAD

**Issue by ywarnier** [chamilo/chamilo-lms#5716]
The global questions bank at /main/admin/questions.php is broken (style broken from the move to the new Twig structure).
Needs fixing.

![questions-bank](https://github.com/user-attachments/assets/69e68830-7750-444a-b604-7ae029a2b686)

**Comment by Coussecousse** [comment]
Hello @ywarnier , 
I'm very interested in the Chamilo-LMS project and would like to start contributing. Since this is my first contribution, I wanted to better understand the issue.
I compared the old version of Chamilo (1.11.x) with version 2 to look for differences between questions. The only change I noticed is the label "nom" in v2 is  "Titre" in 1.11.x (probably "Title" in the English version). Everything seems to work fine for me so far, although it’s my first time using Chamilo.
The only problem I’ve encountered is that the PDF export doesn’t seem to work (probably because of the style as you mentionned ?). Could you clarify what kind of help or contribution you expect on this issue? I’d be happy to work on it. 😄
Thanks in advance!

<img width="1544" height="804" alt="Image" src="https://github.com/user-attachments/assets/17f4a5cc-5792-4982-a497-6c86bf5b189a" />

**Comment by ywarnier** [comment]
You're right, the style seems to be correct now but there are other issues with SQL queries now. I will work on fixing those, then come back to update this in case there are still visualization issues.

**Comment by ywarnier** [comment]
In the results, a preview of the question and answers should be seen (it is in Chamilo 1). For some reason, we have lost the preview (which appears on @Coussecousse 's screenshot).

The rest seems fine.

**Comment by christianbeeznest** [comment]
It is checked and improved with this commit https://github.com/chamilo/chamilo-lms/pull/7189/commits/e0ef1d2f5a04f67355a1963514a431887949b51e

Thanks for confirmation

**Comment by NicoDucou** [comment]
Hi @christianbeeznest 

It now shows the question correctly so this is fine.
I've just seen a bug.
When searching the questions appear then click on a question and then on the edit button.
The link of the edit button is correct in the form of main/exercise/admin.php?cid=2&sid=0&gid=0&exerciseId=0&type=1&editQuestion=1

But we get to the list of exercise in the course : /main/exercise/exercise.php?cid=2&sid=0&gid=0&gradebook=0&origin=

But if we enter a question for edition in the new tab, then get back to the question bank tab and click on the same edit button this time it opens the question edition page correctly, so there is problably something in the context that do not work properly.

To test it you can just open this link : https://testing24.beeznest.com/main/exercise/admin.php?cid=2&sid=0&gid=0&exerciseId=0&type=1&editQuestion=1

If you open this link it will redirect you to the list of exercise in this course.
If you open a question in edition and then open this link then it will show you the question edition page. 

It should always show you the question edition page.

**Comment by christianbeeznest** [comment]
> Hi [@christianbeeznest](https://github.com/christianbeeznest)
> 
..
> 
> It should always show you the question edition page.

It is checked with this commit https://github.com/chamilo/chamilo-lms/pull/7201/commits/b7da18604ef1f695497a48aa8fd89d1e7da293ae

Thanks for confirmation.

**Comment by NicoDucou** [comment]
All good with this task now.
